Marlin/Marlin_main.cpp

```diff
@@ -2238,28 +2238,15 @@ static void clean_up_after_endstop_or_probe_move() {
    *
    * @return The raw Z position where the probe was triggered
    */
-  #define HAS_CALIBRATION_PROBE (ENABLED(DELTA_AUTO_CALIBRATION) && Z_PROBE_LOW_POINT < 0)
```

Comment:
Z_PROBE_LOW_POINT testing is already done in SanetyCheck.h

#if Z_PROBE_LOW_POINT > 0
  #error "Z_PROBE_LOW_POINT must be less than or equal to 0."
#endif